README.md

```diff
@@ -8,52 +8,65 @@ A loading indicator component for EmberJS that animates across the top of the vi
 
 ## Installation
 
-`npm install ember-cli-loading-slider --save-dev`
+`ember install ember-cli-loading-slider`
 
 ## Usage
 
 Add the component to your application template:
 
-    {{loading-slider isLoading=loading duration=250}}
-
-Alternatively you may try the more complex mode. In this mode an array of colors
-is required `hexColorsArray: ['#000', '#fff']`:
-
-    {{loading-slider isLoading=loading expanding=true color=hexColorsArray}}
+    {{loading-slider duration=250}}
```

Comment:
Main reason was this [line in the service PR](https://github.com/jerel/ember-cli-loading-slider/pull/19/files#diff-33efff51085b97215b246f6fcfd9e0feR2).

Additionally I understood the service should be used to control the component state and not the passed in properties anymore while moving forward to use services.

But anyhow I can undo this change or maybe elaborate more the two ways to use the component.